launch/dynamo-run/src/subprocess/trtllm_inc.py

```diff
@@ -7,18 +7,28 @@
 #
 # `dynamo-run out=trtllm` runs this script
 # Can be used standalone: `python3 trtllm_inc.py` - lots of optional cmd line params
+#
+# Disaggregated serving:
+# - Ingress: dynamo run in=http out=dyn
+# - Decode Worker: python3 trtllm_inc.py --disagg-mode=decode --extra-engine-args=trtllm_config/sample.yaml
+# - Prefill Worker: python3 trtllm_inc.py --disagg-mode=prefill --extra-engine-args=trtllm_config/sample.yaml
```

Comment:
Note, compared to old approach that re-use trtllm-serve disagg structure:
```
context:
  ...

generation:
  ...
```

This approach expects you to have separate `prefill.yaml` and `decode.yaml` (if needed)

Reply:
Seems to work out of box for DS R1 after setting separate yaml configs for each prefill/decode - nice! 🚀